wizeline/project-lab
# Feat - 221 - Add last update field to the project detail component

#### What does this PR do?

Adding the last update date, to the project detail component. And fixing a bug where the updatedAt field didn't change when a user update the project details.

#### Where should the reviewer start?

- app/pages/projects/[projectId]/index.tsx
- app/projects/mutations/updateProject.ts

#### How should this be manually tested?

1. Join a project
2. See the last update field in the description box
3. Click the edit project icon
4. Edit the project
5. Click in "Update"
6. See the updateAt field change to the last updated

#### Any background context you want to provide?

Testing the updateAt field I saw that it didn't change and I already fix the bug by adding a change in the mutation/updateProject.ts file.

#### What are the relevant tickets?

Issue - #221 

#### Screenshots

<img width="1432" alt="image" src="https://user-images.githubusercontent.com/97306904/159585193-0fe76b5e-7a5d-4176-b3d3-44579f128e1e.png">
